### PR TITLE
Clean up general config page code

### PR DESCRIPTION
**Pull request by spacegaier**
## Proposed change

The coding still contained lots of stuff that has already been moved over to `ha-config-server-control.ts`. This is now cleaned up.

Plus a little CSS fix for the config checker output. First line was incorrectly handled.

**Before:**

image

**After:**

image

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [X] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/core/ha-config-section-core.js [hunk 1]
```diff
@@ -19,37 +19,6 @@ import "./ha-config-url-form";
 class HaConfigSectionCore extends LocalizeMixin(PolymerElement) {
   static get template() {
     return html`
-      <style include="iron-flex ha-style">
```
Comment (bramkragten) [inline src/panels/config/core/ha-config-section-core.js:22]: We also don't use anything from `ha-style`?

**Review by bramkragten — APPROVED** [review]

File: src/panels/config/core/ha-config-section-core.js (hunk 1 above)
Comment (spacegaier) [reply src/panels/config/core/ha-config-section-core.js:22]: I checked before and after and could not spot any visual difference, so I just assumed "no" :) .

**Comment by spacegaier** [comment]
@bramkragten One thing I just saw is that there is no space between the input field and the suffix slot:

image

Is there a generic way to solve that, not just for this single card?

**Comment by bramkragten** [comment]
> @bramkragten One thing I just saw is that there is no space between the input field with the up/down arrows and the suffix slot ("meters"):
> 
> image
> 
> Is there a generic way to solve that, not just for this single card?

Is that just a default paper-input with type number?

**Comment by spacegaier** [comment]
Yes, it is.

**Pull request merged**